x-pack/platform/packages/shared/kbn-ai-assistant/src/chat/chat_body.tsx

```diff
@@ -413,7 +413,7 @@ export function ChatBody({
 
   const showKnowledgeBaseReIndexingCallout =
     knowledgeBase.status.value?.enabled === true &&
-    knowledgeBase.status.value?.kbState === KnowledgeBaseState.READY &&
+    knowledgeBase.status.value?.inferenceModelState === KnowledgeBaseState.READY &&
```

Comment:
We should also rename the enum:

```suggestion
    knowledgeBase.status.value?.inferenceModelState === InferenceModelState.READY &&
```

If this change causes too many changes in this PR I'm fine with leaving it for a follow up.